.github/actions/setup/action.yml

```
@@ -3,9 +3,7 @@ name: Setup
 runs:
   using: composite
   steps:
-    - uses: actions/setup-node@v3
-      with:
-        node-version: 16.x
+    - uses: actions/setup-node@v4
```

We should still specify node 20 to be more explicit.  setup-node's documentation recommends to "always specify Node.js version and don't rely on the system one" 
```suggestion
    - uses: actions/setup-node@v4
      with:
        node-version: 20.x
```

---

I used `code blame` to see if there was a related comment justifying an explicit version in the past. But forgot to check the docs 😅 

I agree, let's revert it back